app/views/layouts/application.html.slim

```diff
@@ -3,7 +3,11 @@ html
   head
     title Voluntaries Platform
     = stylesheet_link_tag 'application', media: 'all',
-    | 'data-turbolinks-track': 'reload'
+      'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
   body
-    = yield
+    = render 'layouts/nav'
```

Comment:
move this to `views/application` folder, then you just can include it with `= render 'nav'`
And please write out names

Why this works is, rails searches through a folder tree for partials, the last one it looks through is application. So global partials should be in there